### PR TITLE
remove emails of people that left Mirai

### PR DESCRIPTION
**Pull request by Chargothrond**
Do not merge before February.

We still need to define a new person to receive the "cre" role. Should we just add @riccardoporreca as the only person left with an email or someone else? What do you think @RolandASc ?

Another open point is how to bring this to master: I don't think it makes sense to create a new release for this, but we need to update the DESCRIPTION file there as well eventually. Someone with admin access could also change the default branch to "develop", which would anyway be the expectation for the GitFlow approach.

### CODE REVIEW

**Comment by Chargothrond** [comment]
@riccardoporreca @RolandASc , this is now ready for review since it is February. Tbd if we just also merge the branch into master to get the updates on both develop and master.

**Review by RolandASc — APPROVED** [review]

**Comment by RolandASc** [comment]
I wouldn't mind switching the default branch to *develop*, but we'd have to update the README accordingly and should check any implications on the CI.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-                    email = "francesca.vitalini@mirai-solutions.com"),
-             person("Nicoletta", "Farabullini", role = c("aut"),
-                    email = "nicoletta.farabullini@mirai-solutions.com"),
-             person("Riccardo", "Porreca", role = c("ctb"),
+Authors@R: c(person("Gabriel", "Foix", role = c("aut")),
+             person("Francesca", "Vitalini", role = c("aut")),
+             person("Nicoletta", "Farabullini", role = c("aut")),
+             person("Riccardo", "Porreca", role = c("ctb", "cre"),
                     email = "riccardo.porreca@mirai-solutions.com"),
              person("Mirai Solutions GmbH", role = c("ctb", "cph"),
```
Comment (riccardoporreca) [inline DESCRIPTION]: Let's make the maintainer an author, which makes more sense in general (+ the contributor role was assigned very early before I stated "authoring" the app quite a bit).
Also, Mirai should just be the copyright holder

```suggestion
             person("Riccardo", "Porreca", role = c("aut", "cre"),
                    email = "riccardo.porreca@mirai-solutions.com"),
             person("Mirai Solutions GmbH", role = "cph",
```

**Review by riccardoporreca — APPROVED** [review]
LGTM, see a small suggestion.

Let's merge this one to develop and see afterwards what to do next in terms of default branch etc.

**Pull request merged**